low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,42 @@ import sys
 import socket
 import json
 import subprocess
+import warnings
+from framework.utils.conf_utils import (GLOBAL_CONF, Conf,
+    STORAGE_TYPE_KEY, SITE_ID_KEY, RACK_ID_KEY,
+    NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.base.sspl_constants import DATA_PATH
 
-from cortx.utils.conf_store import Conf
-from framework.utils.conf_utils import Utility
-from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                        RESOURCE_HEALTH_VIEW,
-                                        sspl_config_path,
-                                        GLOBAL_CONFIG_INDEX)
+warnings.filterwarnings(action='ignore', module='cryptography')
 
-SSPL_CONF = "SSPL_CONFIG"
-Conf.load(SSPL_CONF, sspl_config_path)
-global_config_url = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>global_config_copy_url")
-Conf.load(GLOBAL_CONFIG_INDEX, global_config_url)
+ManifestData = f"{DATA_PATH}manifest"
+if not os.path.exists(ManifestData):
+    os.makedirs(ManifestData)
 
-machine_id = Utility.get_machine_id()
-
-# Get cluster_id and node_key_id from global config
-cluster_id = Conf.get(GLOBAL_CONFIG_INDEX, "server_node>%s>cluster_id" % machine_id)
-node_key_id = Conf.get(GLOBAL_CONFIG_INDEX, "server_node>%s>name" % machine_id)
-
-onGoingRequestPath = "/tmp/sspl-msb-req.txt"
+onGoingRequestPath = f"{ManifestData}/sspl-msb-req.txt"
```

Comment:
change the variable name to "sspl_msb_req"
and delete that file when the program executed successfully or even if any exception occured.

Reply:
Done.